app/vmui/packages/vmui/src/components/Main/Button/style.scss

```diff
@@ -45,6 +45,10 @@ $button-radius: 6px;
     transform: translateZ(-1px);
   }
 
+  &.prettify:active {
```

Comment:
I believe it's excessive to add a separate `:active` state for just one button. In this case, we can add a general one:

```css
&:active:after {
  transform: scale(0.9);
}
```